README.md

```diff
@@ -240,10 +269,16 @@ Possible extra arguments to pass:
 * `--node` - Run Node.js' own tests with Electron in `RUN_AS_NODE` mode.
 * `--runners=<main|remote|native>` - The set of tests to run, can be either `main`, `remote`, or `native`.
 
-## Getting Information
+## `e show`
```

Comment:
IIRC what I was thinking was grouping the commands by topic, and "Getting Information" would have subsections for `e show` and `e open`, which is why `e open` below is marked up as `###`.

I like this change and am :+1: on it, but it needs to tweak that `### e open` to now be `## e open` so that it's not a subsection of `e show`